tests/test_maps.py

```diff
@@ -44,6 +45,10 @@ def test_setup_map():
     }
     ds.Map(**kwargs).show()
 
+def test_map_feature_casting():
+    """ Tests that a map is succesfully created when features is a numpy array """
+    assert isinstance(np.array([]), np.ndarray)
```

Comment:
I would prefer this test case needs to add some sort of non-empty array (and the use of `make_array` rather than `np.array([])`) to be a proper test case. Thanks!